content/articles/dkim-record.markdown

```diff
@@ -0,0 +1,62 @@
+---
+title: DKIM records
+excerpt: This article explains how DKIM records work.
+categories:
+- DNS
+- Email
+---
+
+# DKIM Records
+
+* TOC
+{:toc}
+
+---
+
+DKIM stands for DomainKeys Identified Email and provides a way to validate that an organization delivering an email has the right to do so.
```

Comment:
Nope, it's actually written that way in the RFCs:

- https://www.ietf.org/rfc/rfc4871.txt
- https://tools.ietf.org/html/rfc6376